_includes/api/en/4x/app-settings.md

```diff
@@ -1,25 +1,41 @@
-If `name` is one of the application settings, it affects the behavior of the application. The following table lists application settings.
+The following table lists application settings.
+
+Note that sub-apps will:
+
+* Not inherit the value of settings that have a default value.  You must set the value in the sub-app.
+* Inherit the value of settings with no default value; these are explicitly noted in the table below.
+
+Exceptions: Sub-apps will inherit the value of `trust proxy` even though it does not have a default value (for backward-compatibility);
```

Comment:
Wrong way round I think : Sub-apps will inherit the value of `trust proxy` even though it **has** a default value